File: PYTHON/BASICS/LEAP.py
def LEAP():
    y=int(input("Enter the year..."))
    if y%400==0:
        print("It is a leap year.")
    elif y%100==0:
        print("It is not a leap year.")
    elif y%4==0:
        print("It is a leap year.")
    else:
        print("It is not a leap year.")

LEAP()

File: C/BASICS/LEAP.c
#include<stdio.h>
int main()
{
    int y;
    printf("Enter the year..");
    scanf("%d",&y);
    if(y%400==0)
    {
        printf("It is a leap year.");
    }
    else if(y%100==0)
    {
        printf("It is not a leap year.");
    }
    else if(y%4==0)
    {
        printf("It is a leap year.");
    }
    else
    {
        printf("It is not a leap year.");
    }
    return 0;
}